src/main/java/ipman/commands/CreateToDoCommand.java

```diff
@@ -0,0 +1,38 @@
+package ipman.commands;
+
+import ipman.models.Task;
+import ipman.models.TaskList;
+import ipman.models.ToDo;
+import ipman.ui.Ui;
+
+/**
+ * Creates a <code>ToDo</code> inside <code>Context</code>'s
+ * <code>TaskList</code>
+ * @see ToDo
+ * @see Context
+ * @see TaskList
+ */
+public class CreateToDoCommand implements Command {
+    private final String name;
+
+    /**
+     * Creates command that creates a particular to-do
+     * @param name to-do name
```

Comment:
Should there be a whitespace between the first line of javadocs and the next line? I mean this was repeated everywhere in the code so I was just wondering.

Reply:
Hmm I think it's not a big deal, but I think it's a good idea to be consistent and have a newline before the [at]s

Reply:
Resolved by https://github.com/nus-cs2103-AY2425S1/ip/pull/538/commits/38439567a10a1f9d6c06742e548b97b801ce2548